packages/desktop-client/src/util/error.ts

```
@@ -0,0 +1,183 @@
+      );
+
+    case 'clock-drift':
+      return t(
+        'Failed to download the budget because your device time differs too much from the server. Please check your device time settings and ensure they are correct.',
+      );
+
+    default:
+      const info =
+        meta && typeof meta === 'object' && 'fileId' in meta && meta.fileId
+          ? `, fileId: ${String(meta.fileId)}`
+          : '';
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)